src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((acc, declaration) => {
```

Comment:
The `split` method is correctly used to separate the CSS declarations, but ensure that empty entries are filtered out before reducing to avoid unnecessary iterations over empty strings.